src/olympia/lib/settings_base.py

```diff
@@ -1782,6 +1782,9 @@ def get_raven_release():
     ('olympia', 'amo'),
 )
 
+FXA_CONTENT_HOST = 'https://stable.dev.lcip.org'
+FXA_OAUTH_HOST = 'https://oauth-stable.dev.lcip.org/v1'
+FXA_PROFILE_HOST = 'https://stable.dev.lcip.org/profile/v1'
```

Comment:
Maybe the default should be prod instead, and these reset in `dev/settings.py` and `settings.py` ? We haven't been super consistent with this.

In any case stage should use prod settings for these - that needs to be fixed.

Reply:
While you're at it, can you get rid of `VALID_LOGIN_REDIRECTS` ? It's used in `devhub/views.py` in `_clean_next_url()` but judging from the contents of that variable it's completely outdated, gotta be obsolete.